po/de.po

```diff
@@ -22339,13 +22327,19 @@ msgid ""
 "It took %.2f seconds to enumerate untracked files,\n"
 "but the results were cached, and subsequent runs may be faster."
 msgstr ""
+"Es dauerte %.2f Sekunden, um die unversionierten Dateien aufzuzählen,\n"
+"aber die Ergebnisse wurden zwischengespeichert, so dass spätere Durchläufe "
```

Comment:
"sodass" is used more often, but I do not care which version we use, unless we are consistent 😉 
https://studyflix.de/deutsch/sodass-oder-so-dass-3794